Add-on Documentation/Performance & Monitoring/Blackfire.md

```diff
@@ -0,0 +1,46 @@
+# Blackfire
+
+Blackfire Profiler automatically instruments your code to gather data about consumed server resources like memory, CPU time, and I/O.
+
+## Adding Blackfire
+
+The Blackfire add-on can be added to every deployment with:
+
+~~~
+$ cctrlapp APP_NAME/DEP_NAME addon.add blackfire.OPTION
+~~~
+
+When added, Blackfire automatically creates an user for the application and updates the addon configuration with its credentials.
+You can access Blackfire for your deployment in the web console via Single Sign-On (SSO).
+Navigate to the specific deployment, choose "Add-Ons" tab, click on "Settings" link and "Login to dashboard".
+
+On Blackfire, Single Sign-On acts as an authorization. You will have to log in (or sign) into an other account and then it will be authorized to see your profiles.
+
```

Comment:
On Blackfire, first log-in with Github, Google+, or SensioLabsConnect. Then, the Single Sign-On link from cloudControl acts as an authorization layer.